optimum/exporters/tasks.py

```diff
@@ -925,8 +949,10 @@ def get_supported_model_type_for_task(task: str, exporter: str) -> List[str]:
         ]
 
     @staticmethod
-    def format_task(task: str) -> str:
-        return task.replace("-with-past", "")
+    def map_from_legacy(task: str) -> str:
+        if task in TasksManager._LEGACY_TASK_MAP:
+            task = TasksManager._LEGACY_TASK_MAP[task]
```

Comment:
Should we log a warning here telling the user the non-legacy task name to use?

Reply:
Done!